examples/kafka-hub/hub/modules/config/configurations.bal

```diff
@@ -68,3 +68,6 @@ public final string WEBSUB_EVENTS_CONSUMER_GROUP = os:getEnv("WEBSUB_EVENTS_CONS
 isolated function constructSystemConsumerGroup() returns string {
     return string `websub-events-receiver-${SERVER_IDENTIFIER}-${util:generateRandomString()}`;
 }
+
+# The client MTLS configurations used by Kafka consumers mapped to HTTP subscribers
+public configurable map<types:KafkaClientKeyStoreConfig> KAFKA_CLIENT_KS_CONFIGS = ?;
```

Comment:
We usually dont use snake case unless its a compile-time constant right? shouldn't we stick to camel case in here?
```suggestion
public configurable map<types:KafkaClientKeyStoreConfig> kafkaClientKSConfigs = ?;
```